packages/components/src/index.html

```diff
@@ -15,13 +15,16 @@
 
   <script defer>
 
+    const myfunc = () => { 
+      console.log('here')
+    }
+
```

Comment:
i hope i can remove this :D

Reply:
No, this piece of code is extremely important to the proper functioning of the entire project :D